airbyte-integrations/connectors/source-my-hours/source_my_hours/manifest.yaml

```diff
@@ -0,0 +1,165 @@
+          type: DpathExtractor
+          field_path: []
+      paginator:
+        type: NoPagination
+  - type: DeclarativeStream
+    name: time_logs
+    primary_key:
+      - logId
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://api2.myhours.com/api
+        path: Reports/activity
+        http_method: GET
+        request_parameters:
+          DateFrom: '{{ config[''start_date''] }}'
+          DateTo: '{{ today_utc() }}'
+        request_headers: {}
+        authenticator:
+          type: BearerAuthenticator
+          api_token: '{{ config[''api_key''] }}'
+        request_body_json: {}
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: []
+      paginator:
+        type: NoPagination
+  - type: DeclarativeStream
+    name: tags
+    primary_key:
+      - id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://api2.myhours.com/api
+        path: Tags
+        http_method: GET
+        request_parameters: {}
+        request_headers: {}
+        authenticator:
+          type: BearerAuthenticator
+          api_token: '{{ config[''api_key''] }}'
+        request_body_json: {}
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: []
+      paginator:
+        type: NoPagination
+  - type: DeclarativeStream
+    name: projects
+    primary_key:
+      - id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://api2.myhours.com/api
+        path: Projects/getAll
+        http_method: GET
+        request_parameters: {}
+        request_headers: {}
+        authenticator:
+          type: BearerAuthenticator
+          api_token: '{{ config[''api_key''] }}'
+        request_body_json: {}
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: []
+      paginator:
+        type: NoPagination
+  - type: DeclarativeStream
+    name: clients
+    primary_key: 
+      - id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://api2.myhours.com/api
+        path: Clients
+        http_method: GET
+        request_parameters: {}
+        request_headers: {}
+        authenticator:
+          type: BearerAuthenticator
+          api_token: '{{ config[''api_key''] }}'
+        request_body_json: {}
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: []
+      paginator:
+        type: NoPagination
+spec:
+  connection_specification:
+    $schema: http://json-schema.org/draft-07/schema#
+    type: object
+    required:
+      - api_key
+      - start_date
+    properties:
+      api_key:
```

Comment:
why are we not using the email+password scheme we were previously using?

I think we should still support the OAuth mechanism that is already implemented. You might need to create a custom authenticator that overwrites `build_refresh_request_body` to create the right payload. Its implementation would be very similar to the [existing retrieve_refresh_token](https://github.com/airbytehq/airbyte/pull/36063/files#diff-6382eba3a528b81c51fcf20ca2d2d0b3fe078246c484d8d9b66349e230d3c775L28-L37).

[Here's an example of a custom authenticator for source-younium](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-younium/source_younium/components.py#L19-L86).

Reply:
As per previous comment
https://github.com/airbytehq/airbyte/pull/36063#discussion_r1525181661

Do we want it back? What's you opinion @girarda

Reply:
I don't think this will work because the access key returned from `https://api2.myhours.com/api/tokens/login` is only valid for 5 days

Reply:
Okay will work on selective auth